docs/source/operations/projections/mod_krovak.rst

```diff
@@ -0,0 +1,77 @@
+.. _mod_krovak:
+
+********************************************************************************
+Krovak Modified
+********************************************************************************
+
+.. versionadded:: 9.4.0
+
++---------------------+----------------------------------------------------------+
+| **Classification**  | Conical                                                  |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forward and inverse, spherical and ellipsoidal           |
++---------------------+----------------------------------------------------------+
+| **Defined area**    | Czech Republic                                           |
++---------------------+----------------------------------------------------------+
+| **Alias**           | mod_krovak                                               |
++---------------------+----------------------------------------------------------+
+| **Domain**          | 2D                                                       |
++---------------------+----------------------------------------------------------+
+| **Input type**      | Geodetic coordinates                                     |
++---------------------+----------------------------------------------------------+
+| **Output type**     | Projected coordinates                                    |
++---------------------+----------------------------------------------------------+
+
+
+.. figure:: ./images/krovak.png
+   :width: 500 px
+   :align: center
+   :alt:   Krovak Modified
```

Comment:
This is a repeat of an existing figure. Does it make sense to create a new `mod_krovak.png`? It would need to be added to docs/plot/plotdefs.json

Reply:
> Does it make sense to create a new `mod_krovak.png`?

Not really. The difference between both is sub-metric...